render-wasm/src/shapes/text.rs

```diff
@@ -565,16 +565,16 @@ impl Paragraph {
     }
 
     #[allow(dead_code)]
-    fn set_children(&mut self, children: Vec<TextLeaf>) {
+    fn set_children(&mut self, children: Vec<TextSpan>) {
         self.children = children;
     }
 
-    pub fn children(&self) -> &[TextLeaf] {
+    pub fn children(&self) -> &[TextSpan] {
         &self.children
     }
 
     #[allow(dead_code)]
-    fn add_leaf(&mut self, leaf: TextLeaf) {
+    fn add_leaf(&mut self, leaf: TextSpan) {
```

Comment:
should we also update here `add_leaf` and `leaf`? same in other parts of this file